tests/matplotlib/test_mpl_scene.py

```diff
@@ -46,3 +48,61 @@ def test_scene_init_with_ax(self):
         fig, ax = plt.subplots(subplot_kw={"projection": "3d"})
         scene = Scene3D(self.rf, self.zp, ax=ax)
         assert scene.axes == ax
+
+    def test_scene_in_orthogonal_proj_no_arguments_view_of_zy_plane_of_rf(self):
+        axes_mock = MagicMock()
+        scene = Scene3D(self.rf, self.zp, ax=axes_mock)
+
+        scene.as_orthogonal_projection_plot()
+
+        kwargs = {
+            "elev": 0.0,
+            "azim": 0.0,
+            "roll": 0.0,
+        }
+        axes_mock.view_init.assert_called_once_with(**kwargs)
+
+    def test_scene_in_orthogonal_proj_rf_as_arguments_view_of_zy_plane_of_rf(self):
+        axes_mock = MagicMock()
+        scene = Scene3D(self.rf, self.zp, ax=axes_mock)
+
+        scene.as_orthogonal_projection_plot(self.rf)
+
+        kwargs = {
+            "elev": 0.0,
+            "azim": 0.0,
+            "roll": 0.0,
+        }
+        axes_mock.view_init.assert_called_once_with(**kwargs)
+
+    def test_scene_in_orthogonal_projection_view_of_xy_plane_of_rf(self):
+        axes_mock = MagicMock()
+        scene = Scene3D(self.rf, self.zp, ax=axes_mock)
+
+        A = me.ReferenceFrame("A")
+        A.orient_axis(self.rf, -self.rf.y, np.pi / 2)
+
+        scene.as_orthogonal_projection_plot(A)
+
+        kwargs = {
+            "elev": 90.0,
+            "azim": 0.0,
+            "roll": 0.0,
+        }
+        axes_mock.view_init.assert_called_once_with(**kwargs)
+
```

Comment:
```suggestion
        A.orient_axis(self.rf, self.rf.y, np.pi / 2)

        scene.as_orthogonal_projection_plot(A)

        kwargs = {
            "elev": 90.0,
            "azim": -90.0,
            "roll": 0.0,
        }
        axes_mock.view_init.assert_called_once_with(**kwargs)
```
It might be a little counterintuitive, but you are viewing the YZ plane of A, this implies that you have to do the opposite rotation of what you were currently doing to get the XY plane (another option is to use `self.rf.orient_axis(A, -A.y, np.pi / 2)`).

P.S. [This page](https://matplotlib.org/stable/api/toolkits/mplot3d/view_angles.html) shows that the answer should be `(90, -90, 0)`.

Reply:
it took me a while, I have a solution that I understand, although it do find a bit of an odd one.